script/Ciel/Day 7/07_01_CIEL07_2_2.txt

```diff
@@ -665,7 +665,7 @@ I can't help but be reminded of the sensation I felt back when we were pressed u
 // Not sure I agree on standardizing 人でなし, it's useful to be able to inflect the word to fit the context.
 // E.g. here, I'm assuming it's more about him feeling the frisk than him being an asshole?
 "%{i}Hehe%{/i}, for someone so fearless when up against vampires, you sure find this scary.
- You're well-off, kind, goodhearted――but underneath all that, you're still a bad boy.
+ You're well-off, kind, goodhearted――but underneath all that, you're still a psychopath.
```

Comment:
You know, this makes way more sense in context now given Noel's freakout at him in the catacombs.